### PR TITLE
change to object and added note about capitalization

### PR DESCRIPTION
**Pull request by newtewt**
#496 
#495

### CODE REVIEW

File: content/en/apps/reference/app-settings/forms.md [hunk 1]
```diff
@@ -10,9 +10,9 @@ relevantLinks: >
 keywords: workflows interop json-forms
 ---
 
-JSON forms are defined in either the `base_settings.json` or the `app_settings/forms.json` file and compiled in to the `app_settings.json` file with the `compile-app-settings` action in the `medic-conf` tool. JSON Forms are used for parsing reports from formatted SMS, SIM applications, and Medic Collect. JSON form definitions are also used for interoperability with third-party systems. Each form is defined as an JSON form object according to the following schema. 
+JSON forms are defined in either the `base_settings.json` or the `app_settings/forms.json` file and compiled in to the `app_settings.json` file with the `compile-app-settings` action in the `medic-conf` tool. JSON Forms are used for parsing reports from formatted SMS, SIM applications, and Medic Collect. JSON form definitions are also used for interoperability with third-party systems. Each form is defined as an JSON form object according to the following schema. The key for each object is required to be capitalized.
```
Comment (abbyad) [inline content/en/apps/reference/app-settings/forms.md:13]: Would be good to clarify that it is the entire string that needs to be uppercase, and I also think uppercase will reasonate more than capitalized to say that. 

```suggestion
JSON forms are defined in either the `base_settings.json` or the `app_settings/forms.json` file and compiled in to the `app_settings.json` file with the `compile-app-settings` action in the `medic-conf` tool. JSON Forms are used for parsing reports from formatted SMS, SIM applications, and Medic Collect. JSON form definitions are also used for interoperability with third-party systems. Each form is defined as an JSON form object according to the following schema. The key for each object must be unique and all characters must be uppercase. 
```

**Review by abbyad — CHANGES_REQUESTED** [review]
Good to have these changes included!

Suggestion inline, but it may be a matter of preference. If you feel strongly I am happy to approve as is.

**Comment by newtewt** [comment]
@abbyad  I think your suggestion makes sense. I committed and just need an approval.

**Review by abbyad — APPROVED** [review]
Looks great, thanks for making the corrections that will surely save someone headaches and time down the line!

**Pull request merged**